environment/src/test/java/com/sequenceiq/environment/network/v1/converter/AwsEnvironmentNetworkConverterTest.java

```diff
@@ -203,23 +213,91 @@ void testSetProviderSpecificNetworkShouldPopulateTheExistingNetworkWithTheNewNet
         assertEquals(SUBNET_CIDR_1, awsNetwork.getSubnetMetas().get(SUBNET_1).getCidr());
         assertFalse(awsNetwork.getSubnetMetas().get(SUBNET_1).isPrivateSubnet());
         assertThat(awsNetwork.getSubnetMetas().get(SUBNET_1).getDeploymentRestrictions())
-                .containsExactlyElementsOf(DeploymentRestriction.ENDPOINT_ACCESS_GATEWAYS);
+                .containsAll(DeploymentRestriction.ALL);
 
         assertEquals(SUBNET_2, awsNetwork.getSubnetMetas().get(SUBNET_2).getId());
         assertEquals(SUBNET_2, awsNetwork.getSubnetMetas().get(SUBNET_2).getName());
         assertEquals(AZ_2, awsNetwork.getSubnetMetas().get(SUBNET_2).getAvailabilityZone());
         assertEquals(SUBNET_CIDR_2, awsNetwork.getSubnetMetas().get(SUBNET_2).getCidr());
         assertFalse(awsNetwork.getSubnetMetas().get(SUBNET_2).isPrivateSubnet());
         assertThat(awsNetwork.getSubnetMetas().get(SUBNET_3).getDeploymentRestrictions())
-                .containsExactlyElementsOf(DeploymentRestriction.ENDPOINT_ACCESS_GATEWAYS);
+                .containsAll(DeploymentRestriction.ALL);
 
         assertEquals(SUBNET_3, awsNetwork.getSubnetMetas().get(SUBNET_3).getId());
         assertEquals(SUBNET_3, awsNetwork.getSubnetMetas().get(SUBNET_3).getName());
         assertEquals(AZ_3, awsNetwork.getSubnetMetas().get(SUBNET_3).getAvailabilityZone());
         assertEquals(SUBNET_CIDR_3, awsNetwork.getSubnetMetas().get(SUBNET_3).getCidr());
         assertFalse(awsNetwork.getSubnetMetas().get(SUBNET_3).isPrivateSubnet());
         assertThat(awsNetwork.getSubnetMetas().get(SUBNET_3).getDeploymentRestrictions())
-                .containsExactlyElementsOf(DeploymentRestriction.ENDPOINT_ACCESS_GATEWAYS);
+                .containsAll(DeploymentRestriction.ALL);
+    }
+
+    @Test
```

Comment:
We can change it to ParameterizedTest if possible. That will make it more readable.